chapter_5/exercise_5_06/getline.c

```diff
@@ -1,23 +1,19 @@
 #include <stdio.h>
 
-void get_line(char *s);
 
-int main(void)
-{
-  char string[150] = "";
+char *getln(char *line);
```

Comment:
@kyopa I don't see the point in returning a pointer to the line. The line is already passed as a function pointer parameter, so why return it if you modify directly the memory already?